### ISSUE TITLE
Form group on the “My Language” screen is not programmatically associated with its group label [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#281]
The “Choose your language” form group label is not announced upon focusing a control in the form group. For example, the “English” checkbox is announced as “English, selected, English, selected checkbox”. 

**Note:** "english, selected" is announced twice by TalkBack.

![settings-Languages - Label - Group language label](https://user-images.githubusercontent.com/7034772/92067031-5583a680-ed71-11ea-89b5-6a1f1c82993e.png)

**Remediation Guidance**

Use the `accessibilityLabel` property to include the group label in the accessible name of each control. Remove the group label from the reading order using the `accessibilityElementsHidden`, as it is redundant when included in the `accessibilityLabel` of each checkbox.

```
<View accessibilityElementsHidden={true}>
  <Text>Choose your language:</Text>
</View>
```
```
<TouchableOpacity
  onPress={…}
  accessibilityLabel='English, Choose your language?'
  accessibilityRole='checkbox'
  accessibilityState={{selected: true}}
/>
```

**Comment by floridemai** [comment]
@jonnyjames This has now been addressed, with the same mention as in https://github.com/project-vagabond/covid-green-app/issues/272. Please re-assign and I'll update.

**Comment by jonnyjames** [comment]
Confirming this issue is fixed. Closing this issue.
